apps/api/src/app/plugins/orm-repositories.ts

```diff
@@ -0,0 +1,67 @@
+import 'reflect-metadata';
+import { FastifyInstance } from 'fastify';
+import typeORMPlugin from 'typeorm-fastify-plugin';
+import fp from 'fastify-plugin';
+import { resolve } from 'path';
+import { logger } from '@cowprotocol/shared';
+
+import { getDatabaseParams } from '@cowprotocol/repositories';
+import { readdir } from 'fs/promises';
+
+export default fp(async function (fastify: FastifyInstance) {
+  const isProduction = process.env.NODE_ENV === 'production';
+
+  const migrationsDir = resolve(
+    __dirname,
+    '../../../../../libs/repositories/src/migrations'
+  );
```

Comment:
It took me a bit to arrive here, when we run the API using nx, it depends on the build. This PR adds a build step for the repositories too.  This way, we can find the migrations and use them.